modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/navigate.es.js

```diff
@@ -32,7 +32,25 @@ export default function (url, listeners) {
 			});
 		}
 	}
-	else {
+	else if (_isValidURL(url)) {
 		window.location.href = url;
 	}
 }
+
+function _isValidURL(url) {
```

Comment:
We don't need to mark functions as "private" by using an underscore. The function is private to the module no matter what it is called so there is no need to worry about somebody accidentally accessing something they are not supposed to.

Reply:
Removed the underscore from the name of the function. Thanks for the feedback!